plugins/inputs/net_response/net_response.go

```diff
@@ -87,18 +92,21 @@ func (n *NetResponse) TcpGather() (map[string]interface{}, error) {
 		responseTime = time.Since(start).Seconds()
 		// Handle error
 		if err != nil {
-			fields["string_found"] = false
+			fields["result_type"] = "string_mismatch"
```

Comment:
I know I told you to have this be string_mismatch, but I was wrong...it should be read_failed.  Sorry.